### PR TITLE
chore: update ipfs

### LINKED ISSUE #186: IPFS dependencies cause Yarn warnings
This warning is output when installing `@jspm/generator`:

```
➤ YN0061: │ multiaddr@npm:10.0.1 is deprecated: This module is deprecated, please upgrade to @multiformats/multiaddr
➤ YN0061: │ multiaddr-to-uri@npm:8.0.0 is deprecated: This module is deprecated, please upgrade to @multiformats/multiaddr-to-uri
➤ YN0002: │ dns-over-http-resolver@npm:1.2.3 doesn't provide node-fetch (p89343), requested by native-fetch
➤ YN0002: │ ipfs-grpc-client@npm:0.9.4 doesn't provide google-protobuf (p11141), requested by @improbable-eng/grpc-web
```

It looks as if IPFS client 0.9.2 has moved on to at least avoid the first two warnings, though I'm not sure about the latter two. Might be worth trying to upgrade, or even making the IPFS functionality optional since I imagine it's not super commonly used?

### PR DESCRIPTION
**Pull request by Aslemammad**
Resolves #186

### CODE REVIEW

**Comment by guybedford** [comment]
Looks like the tests just need updating:

```
+ 'https://ga.jspm.io/npm:lit@2.4.1/index.js'
- 'https://ga.jspm.io/npm:lit@2.3.1/index.js'
    at file:///home/runner/work/generator/generator/test/api/update.test.js:54:10 {
```

Also, please sign the digital CLA per the comment.

For IPFS testing, we likely will need to uncomment that section in the workflow, it seems I removed it in the Chomp PR https://github.com/jspm/generator/pull/101.

**Comment by Aslemammad** [comment]
Ipfs works properly now. Both in my system, and also I added it in the CI.

**Comment by Aslemammad** [comment]
Hey Guy, sure, I'm going to work on it.

**Comment by Aslemammad** [comment]
@guybedford Green now.

File: test/node/ipfs.test.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import 'abort-controller/polyfill.js'
```
[the remark this answers is not in the document]
Comment (Aslemammad) [reply test/node/ipfs.test.js:1]: Yes, we can do that!

[the remark this answers is not in the document]
Comment (Aslemammad) [reply test/node/ipfs.test.js:1]: One thing, is that we can remove it as a dep? so only users who use ipfs face an error if they haven't installed it? how's that.

**Comment by Aslemammad** [comment]
@guybedford How can we resolve the `lit` issue in the ci?

**Comment by guybedford** [comment]
What's the CI issue?

**Comment by Aslemammad** [comment]
Oh, i see the ci is updated now, ok, nothing for now.

**Comment by Aslemammad** [comment]
Green now.

**Pull request merged**